File: backend/identity-service/src/main/java/com/isms/identity/dto/response/LoginResponse.java
package com.isms.identity.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class LoginResponse {
	private String accessToken;
	private String refreshToken;
	@Builder.Default
	private String tokenType = "Bearer";
	private Long expiresIn;
	private UserResponse user;
}

File: frontend/isms-app/src/app/models/login-response.ts
export interface LoginResponse {
    success: boolean;
    message: string;
    timestamp: string;
    data: loginData;
}

export interface loginData {
    accessToken: string;
    refreshToken: string;
    tokenType: string;
    expiresIn: number;
    user: UserResponse;

}
export interface UserResponse {
    email: string;
    firstName: string;
    lastName: string;
    maskedPhoneNumber: string;
    emailVerified: boolean;
    lastLoginAt: string;
}
